contrib/local/setup.sh

```diff
@@ -54,6 +54,8 @@ sed -i 's/timeout_propose = "3s"/timeout_propose = "1s"/g' $HOME/.persistenceCor
 sed -i 's/index_all_keys = false/index_all_keys = true/g' $HOME/.persistenceCore/config/config.toml
 
 echo "Update genesis.json file with updated local params"
+sed -i 's/stake/uxprt/g' $HOME/.persistenceCore/config/genesis.json
```

Comment:
i would add this after the init command itself

Reply:
i wanted to keep the bins for updating separate. All updates to genesis.json should happen together was the idea.

Reply:
hmm, it's okay if it works. I would expect `gentx` to fail if bondDenom is not `uxprt` and you are staking `uxprt`